package/MDAnalysis/coordinates/PDB.py

```diff
@@ -493,6 +495,10 @@ class PDBWriter(base.WriterBase):
     are not set (:code:`None` or :code:`np.zeros(6)`,
     see Issue #2698).
 
+    When atom's record_type attribute is present (i.e. Universe object was
+    created by loading a PDB file), ATOM and HETATM record type keywords
```

Comment:
We can add the links here:
```suggestion
    created by loading a PDB file), ATOM_ and HETATM_ record type keywords
```

Reply:
Done